application/backend/app/db/schema.py

```diff
@@ -49,9 +49,9 @@ class PipelineDB(Base):
     is_running: Mapped[bool] = mapped_column(Boolean, default=False)
     data_collection_policies: Mapped[list] = mapped_column(JSON, nullable=False, default=list)
 
-    sink = relationship("SinkDB", uselist=False)
-    source = relationship("SourceDB", uselist=False)
-    model_revision = relationship("ModelRevisionDB", uselist=False)
+    sink = relationship("SinkDB", uselist=False, lazy="joined")
+    source = relationship("SourceDB", uselist=False, lazy="joined")
+    model_revision = relationship("ModelRevisionDB", uselist=False, lazy="joined")
```

Comment:
What's the reason for using `lazy="joined"` instead of the default `lazy='select'`?

Reply:
With `lazy='select'`, `Pipeline.model_validate(pipeline_db)` keeps child entity attrs (`source`, `sink` and `model_revision`) as `None`. Enabling eager loading with `lazy='joined'` fixes that.